quartz/components/styles/search.scss

```diff
@@ -141,6 +142,10 @@
               0 14px 50px rgba(27, 33, 48, 0.12),
               0 10px 30px rgba(27, 33, 48, 0.16);
           }
+
+          a.internal {
+            background-color: none;
+          }
```

Comment:
why do we do this on mobile?

Reply:
Oh if we don't then it will show link hover within the modal.

Reply:
<img width="183" alt="Screenshot 2024-02-01 at 19 05 53" src="https://github.com/jackyzha0/quartz/assets/29749331/5e0b36c7-c303-49cb-b0c8-50177dba33f5">

This has to do with our links to headers are considered `internal` element here.

Actually we can gate this further with `a.internal.alias`

<img width="238" alt="Screenshot 2024-02-01 at 19 06 06" src="https://github.com/jackyzha0/quartz/assets/29749331/354a689e-1374-421a-b061-bb10375ff7f2">